packages/web/index.tsx

```diff
@@ -1,3 +1,5 @@
+/* This points directly to src/index instead of "@draftbit/ui" because we don't want to rely
+on the package.json "main" */
```

Comment:
Would be helpful to explain _why_ we don't want to rely on the package.json main?

Reply:
Oh, good point thought was obvious. Updated!